docs/sample.yml

```diff
@@ -376,6 +445,37 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  
+  #Tak
+  /courses/{id}/user:
+    get:
+      tags:
+        - courses
+      description: Get all user-ids that a course has.
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Status"
+                type: array  
+              example:
+                - course_id: 442c16a8a353
```

Comment:
how about using `DPCS100#1, DPCS101#1` for course_id

Reply:
it's user_id.
Typo!